build-system/tasks/yarn-check.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const exec = require('../exec').exec;
+const getStderr = require('../exec').getStderr;
+const getStdout = require('../exec').getStdout;
+const gulp = require('gulp-help')(require('gulp'));
+const util = require('gulp-util');
+
+
+/**
+ * Does a yarn check on node_modules, and if it is outdated, runs yarn.
+ */
+function yarnCheck() {
```

Comment:
Should the task now be updated to reflect it no longer just checks.

Reply:
Done.